malcolm/modules/pmac/__init__.py

```diff
@@ -1 +1,2 @@
 from . import parts, infos, util
+from .velocityprofile import VelocityProfile
```

Comment:
Could you put VelocityProfile in util please?

Reply:
I see that you have a strict structure, but feel this makes a complex bit of code a little less legible.

Precedent: pandablocksclient.py ?